hr_timesheet_holiday/__openerp__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Link holidays to analytic accounts',
+    'version': '8.0.1.0.0',
+    'category': 'Generic Modules/Human Resources',
+    'summary': "Link holidays to analytic accounts",
+    'author': "Therp BV, Odoo Community Association (OCA)",
```

Comment:
Your company is the author. If you want to mention Therp too, that's great, but at second place. Think hard when filling in your company name, because this will be used as grouping key in the app store and for your statistics